elastic/cloudbeat
# Support cloudbeat running outside k8s

**What does this PR do?**
Supports Cloudbeat running outside a k8s cluster, locally or on EC2, without crashing.

**Checklist**
- [ ] I have added the necessary README/documentation (if appropriate)

How to test:
1. Build cloudbeat binary - `GOOS=darwin go build -v`
2. Run binary - `./cloudbeat -e`

Expected result:
Outside k8s cluster - cloudbeat doesn't crash
Inside k8s - collect and send findings as expected
